src/preferences/dialog/dlgprefvinyl.cpp

```diff
@@ -81,7 +81,7 @@ DlgPrefVinyl::DlgPrefVinyl(QWidget * parent, VinylControlManager *pVCMan,
 
     TroubleshootingLink->setText(coloredLinkString(
             m_pLinkColor,
-            QStringLiteral("Troubleshooting"),
+            "Troubleshooting",
```

Comment:
Why is that?

Reply:
I don't know why.

Reply:
Force pushed https://github.com/mixxxdj/mixxx/pull/4751/commits/0b7f3527017a62f769a035bfddbd695a64725396 with a comment.